assets/style.css

```diff
@@ -529,7 +529,7 @@ _:-ms-fullscreen,
   display: flex;
   flex-wrap: wrap;
   flex-flow: row wrap;
-  justify-content: center;
+  /*justify-content: center;*/
```

Comment:
Er det en grunn til at denne ble fjernet? Jeg ble litt opphengt i at det ikke var sentrert. Syns du det ser bedre ut når det er venstrestilt?

Reply:
åh, den trengte jeg ikke ta vekk nei.. Gjorde det før jeg bestemte meg for å ta med aliasene, da det var kun ett bilde under så det bedre ut med venstrestilt..

Reply:
Skjønner! Jeg legge den tilbake jeg?

Reply:
Done